aion/web/elm/src/Update.elm

```diff
@@ -27,7 +28,25 @@ update msg model =
             { model | user = response } ! []
 
         OnQuestionCreated response ->
-            model ! []
+            case response of
+                RemoteData.Success responseData ->
```

Comment:
Actually I'm not sure what's the point of using RemoteData since we are ignoring all of its privileges. The main cause for using it is the fact that we can distinguish between having loaded empty or not yet having loaded data etc. Seeing how we deal with this everytime makes me cringe and rethink if we should really use it at all if that's the way we're using it. I mean, we can always say that we will implement rest of the pattern checks later, but we both know it's one of the lowest priority things.